src/homedelivery/query.js

```diff
@@ -0,0 +1,84 @@
+// @flow
+import { fetchConfig } from './../lib/config'
+import type {Config} from './../lib/config'
+import {Zuora} from './../lib/Zuora'
+import type {Query} from './../lib/Zuora'
+import moment from 'moment'
+import type { input } from '../querier'
+
+async function queryZuora (deliveryDate, config: Config) {
+  const formattedDate = deliveryDate.format('YYYY-MM-DD')
+  const deliveryDay = deliveryDate.format('dddd')
+  const zuora = new Zuora(config)
+  const subsQuery: Query =
+    {
+      'name': 'Subscriptions',
+      'query': `
+        SELECT
+            RateplanCharge.quantity,
+            Subscription.Name,
+              SoldToContact.Address1,
+              SoldToContact.Address2,
+              SoldToContact.City, 
+              SoldToContact.Country, 
+              SoldToContact.FirstName, 
+              SoldToContact.LastName, 
+              SoldToContact.PostalCode, 
+              SoldToContact.State,
+              SoldToContact.workPhone,
+              Account.SpecialDeliveryInstructions__c
```

Comment:
Change to move delivery instructions back to SoldTo contact: https://github.com/guardian/membership-common/pull/510